contracts/feeds/PriceFeeds.sol

```diff
@@ -62,18 +60,15 @@ contract PriceFeeds is Constants, Ownable {
             WEI_PRECISION;
     }
 
-    //// NOTE: This function returns 0 during a pause, rather than a revert. Ensure calling contracts handle correctly. ///
     function queryReturn(
         address sourceToken,
         address destToken,
         uint256 sourceAmount)
         public
         view
+        pausable
```

Comment:
@Drypto13 so now this will revert instead of returning zero if pricing pause. are there any issues with this?

Reply:
I dont think so. We should never be pausing the price feed contracts anyways

Reply:
@Drypto13 shall we remove the pausable then?

Reply:
It is fine to keep it pausable